src/modules/lists/__tests__/lists.controller.spec.ts

```diff
@@ -100,13 +101,11 @@ describe('modules/lists/ListsController', () => {
 
   describe('myLists', () => {
     let userId: string;
-    let listDto: ListDto;
     let request: any;
-    let response: any;
     const testUserList = [
       {
         public: true,
-        isFavourite: false,
+        isFavorite: false,
```

Comment:
I like the Canadian spelling better... :)

Reply:
this is what has already been defined in our models and I just fixed a typo here